tsconfig.json

```diff
@@ -10,6 +10,7 @@
     { "path": "packages/header-parser" },
     { "path": "packages/publisher" },
     { "path": "packages/typescript-versions" },
```

Comment:
this needs to be updated when typescript-versions is updated, right? Can you add that to the instructions in typescript-versions/index.ts?

Reply:
The list of packages in `package.json` does, yes, though there's a test that enforces that it's up to date so whoever's bumping will definitely know that they forgot. I'll go and add a note, though.

Reply:
Done.